### ISSUE TITLE
Cannot access LAN of wireguard peer A from another peer B

### ISSUE THREAD

**Issue by eirinaios** [wg-easy/wg-easy#1254]
Hi everyone. I've successfully installed wg-easy on a linux vm (LAN IP 192.168.2.0/24) and I'm able to access from my laptop the internal LAN of the wireguard server (the linux VM). My WG IP address subnet is 10.8.0.0/24. What I'd like to do though is to access the LAN (192.168.210.0/24) of a second peer (a mikrotik router) with WG IP 10.8.0.2, but I cannot do so. Should I create specific routes inside the docker container? My wg0.conf is:

```
# Note: Do not edit this file directly.
# Your changes will be overwritten!

# Server
[Interface]
PrivateKey = XXX
Address = 10.8.0.1/24
ListenPort = 51820
PreUp = 
PostUp =  iptables -t nat -A POSTROUTING -s 10.8.0.0/24 -o eth0 -j MASQUERADE; iptables -A INPUT -p udp -m udp --dport 51820 -j ACCEPT; iptables -A FORWARD -i wg0 -j ACCEPT; iptables -A FORWARD -o wg0 -j ACCEPT; 
PreDown = 
PostDown =  iptables -t nat -D POSTROUTING -s 10.8.0.0/24 -o eth0 -j MASQUERADE; iptables -D INPUT -p udp -m udp --dport 51820 -j ACCEPT; iptables -D FORWARD -i wg0 -j ACCEPT; iptables -D FORWARD -o wg0 -j ACCEPT; 

# Client: mikrotik (XXX)
[Peer]
PublicKey = XXX
PresharedKey = XXX
AllowedIPs = 10.8.0.2/32

# Client: macbook (XXX)
[Peer]
PublicKey = XXX
PresharedKey = XXX
AllowedIPs = 10.8.0.3/32
```

The configuration file of my macbook peer is:
`[Interface]
PrivateKey = XXX
Address = 10.8.0.3/24
DNS = 1.1.1.1

[Peer]
PublicKey = XXX
PresharedKey = XXX
AllowedIPs = 10.8.0.0/24, 192.168.210.0/24
Endpoint = YYY
PersistentKeepalive = 25`

Any help you can provide is greatly appreciated!

**Comment by manprinsen** [comment]
Any updates on this?

Is it still not possible to set multiple subnets for AllowedIPs on SERVER?
Eg
`
[Interface]
PrivateKey = <server's private key>
Address = 10.10.1.3/24
ListenPort = 51820

[Peer]
#PeerA
PublicKey = <peer A's public key>
AllowedIPs = 10.10.1.1/32, 192.168.1.0/24

[Peer]
#PeerB
PublicKey = <peer B's public key>
AllowedIPs = 10.10.1.2/32
`

Seems like setting multiple subnets for the environment variable AllowedIPs only affect the wg clients config......

**Comment by JexSrs** [comment]
> But editing Allowed IPs for specific Peers is currently not supported

Are there any plans for this be supported? After skimming the code a little, I understood that by changing this [line](https://github.com/wg-easy/wg-easy/blob/2bd1dffe7e8ace1426a174608be44a708268bd0c/src/lib/WireGuard.js#L123) in the `WireGuard.js` from:

```
AllowedIPs = ${client.address}/32
```
to
```
AllowedIPs = ${client.allowedIPs}
```
would allow us to set our own IPs.

**Comment by manprinsen** [comment]
Thanks! I'll try it out tomorrow :)

**Comment by elix1er** [comment]
bump

**Comment by kaaax0815** [comment]
Closing because of #1482 

Implemented in #1333
